src/geometry/CTiglBSplineApproxInterp.h

```diff
@@ -51,7 +51,7 @@ struct CTiglApproxResult
 class CTiglBSplineApproxInterp
 {
 public:
-    TIGL_EXPORT CTiglBSplineApproxInterp(const TColgp_Array1OfPnt& points, int nControlPoints, int degree=3);
+    TIGL_EXPORT CTiglBSplineApproxInterp(const TColgp_Array1OfPnt& points, int nControlPoints, int degree=3, bool continuous_if_closed=false);
```

Comment:
Please use a more C++ style name for this argument. E.g. "periodic" (this is also, what the opencascade interpolations are using). A additional docstring can explain, what this option is doing,